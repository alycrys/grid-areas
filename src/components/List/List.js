import React from "react";
import Item from "../Item";

const List = ({ className, items }) => {
  return (
    <div className={className}>
      {items.map(item => {
        return <Item key={item.id} item={item} />;
      })}
    </div>
  );
};

export default List;
