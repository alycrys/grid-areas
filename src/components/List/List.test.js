import React from "react";
import renderer from "react-test-renderer";
import List from "./List";
import items from "../../mocks/data";

it("renders correctly", () => {
  const wrapper = renderer.create(<List items={items} />);

  expect(wrapper.toJSON().children.length).toBe(15);
  expect(wrapper.toJSON()).toMatchSnapshot();
});
