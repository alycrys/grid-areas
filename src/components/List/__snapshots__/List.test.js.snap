// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="item"
  >
    <img
      alt="Lorem Ipsum"
      src="https://placehold.it/1920x1080"
    />
    <div
      className="copy"
    >
      <div
        className="title"
      >
        Lorem Ipsum
      </div>
      <div
        className="content"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed lectus lacus, maximus nec ornare a, volutpat vel lorem. Duis erat ex, placerat in cursus vel, posuere ac dui. Vivamus placerat lorem felis, vel tempor leo imperdiet nec. Mauris purus purus, pharetra vitae malesuada vel, mollis sit amet odio. Nam tincidunt nisl ac turpis scelerisque, sed facilisis dui fermentum. Aliquam fermentum cursus mi ut euismod. Phasellus vel convallis orci, vel tempor elit. Vivamus ut ipsum elit. Ut sollicitudin scelerisque ex.Morbi suscipit, nisl id fermentum consectetur, est risus condimentum nulla, bibendum faucibus augue risus ac odio. In a urna tincidunt, scelerisque diam a, dapibus arcu. Proin consequat lectus nec orci venenatis, vel vestibulum dolor rutrum. In velit felis, volutpat eu suscipit eget, congue at ipsum. Integer libero urna, pretium id pretium ac, rhoncus in arcu. Integer vestibulum erat sed dui aliquam, ac gravida neque facilisis. Donec mattis ligula nibh, ac vestibulum dolor consequat ac.Fusce ultricies enim in cursus scelerisque. Pellentesque ut augue eu erat volutpat finibus. Sed at malesuada arcu, id hendrerit lorem. Nulla accumsan vitae sapien at vestibulum. Cras luctus accumsan commodo. Praesent dictum mauris nec nunc varius dapibus. Suspendisse venenatis ante maximus leo ornare tincidunt. Nunc vulputate eget orci vel facilisis. Aenean vitae viverra quam, ut porta dui. Donec volutpat ex odio, ut malesuada nulla mattis at. Mauris elementum, nulla eu pharetra consectetur, tellus libero sagittis ligula, id fringilla lectus purus id purus. Suspendisse et felis diam. Interdum et malesuada fames ac ante ipsum primis in faucibus. Proin vitae mi vel sapien fermentum fermentum sed nec odio.
      </div>
    </div>
  </div>
  <div
    className="item"
  >
    <img
      alt="Lorem Ipsum"
      src="https://placehold.it/1920x1080"
    />
    <div
      className="copy"
    >
      <div
        className="title"
      >
        Lorem Ipsum
      </div>
      <div
        className="content"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed lectus lacus, maximus nec ornare a, volutpat vel lorem. Duis erat ex, placerat in cursus vel, posuere ac dui. Vivamus placerat lorem felis, vel tempor leo imperdiet nec. Mauris purus purus, pharetra vitae malesuada vel, mollis sit amet odio. Nam tincidunt nisl ac turpis scelerisque, sed facilisis dui fermentum. Aliquam fermentum cursus mi ut euismod. Phasellus vel convallis orci, vel tempor elit. Vivamus ut ipsum elit. Ut sollicitudin scelerisque ex.Morbi suscipit, nisl id fermentum consectetur, est risus condimentum nulla, bibendum faucibus augue risus ac odio. In a urna tincidunt, scelerisque diam a, dapibus arcu. Proin consequat lectus nec orci venenatis, vel vestibulum dolor rutrum. In velit felis, volutpat eu suscipit eget, congue at ipsum. Integer libero urna, pretium id pretium ac, rhoncus in arcu. Integer vestibulum erat sed dui aliquam, ac gravida neque facilisis. Donec mattis ligula nibh, ac vestibulum dolor consequat ac.Fusce ultricies enim in cursus scelerisque. Pellentesque ut augue eu erat volutpat finibus. Sed at malesuada arcu, id hendrerit lorem. Nulla accumsan vitae sapien at vestibulum. Cras luctus accumsan commodo. Praesent dictum mauris nec nunc varius dapibus. Suspendisse venenatis ante maximus leo ornare tincidunt. Nunc vulputate eget orci vel facilisis. Aenean vitae viverra quam, ut porta dui. Donec volutpat ex odio, ut malesuada nulla mattis at. Mauris elementum, nulla eu pharetra consectetur, tellus libero sagittis ligula, id fringilla lectus purus id purus. Suspendisse et felis diam. Interdum et malesuada fames ac ante ipsum primis in faucibus. Proin vitae mi vel sapien fermentum fermentum sed nec odio.
      </div>
    </div>
  </div>
  <div
    className="item"
  >
    <img
      alt="Lorem Ipsum"
      src="https://placehold.it/1920x1080"
    />
    <div
      className="copy"
    >
      <div
        className="title"
      >
        Lorem Ipsum
      </div>
      <div
        className="content"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed lectus lacus, maximus nec ornare a, volutpat vel lorem. Duis erat ex, placerat in cursus vel, posuere ac dui. Vivamus placerat lorem felis, vel tempor leo imperdiet nec. Mauris purus purus, pharetra vitae malesuada vel, mollis sit amet odio. Nam tincidunt nisl ac turpis scelerisque, sed facilisis dui fermentum. Aliquam fermentum cursus mi ut euismod. Phasellus vel convallis orci, vel tempor elit. Vivamus ut ipsum elit. Ut sollicitudin scelerisque ex.Morbi suscipit, nisl id fermentum consectetur, est risus condimentum nulla, bibendum faucibus augue risus ac odio. In a urna tincidunt, scelerisque diam a, dapibus arcu. Proin consequat lectus nec orci venenatis, vel vestibulum dolor rutrum. In velit felis, volutpat eu suscipit eget, congue at ipsum. Integer libero urna, pretium id pretium ac, rhoncus in arcu. Integer vestibulum erat sed dui aliquam, ac gravida neque facilisis. Donec mattis ligula nibh, ac vestibulum dolor consequat ac.Fusce ultricies enim in cursus scelerisque. Pellentesque ut augue eu erat volutpat finibus. Sed at malesuada arcu, id hendrerit lorem. Nulla accumsan vitae sapien at vestibulum. Cras luctus accumsan commodo. Praesent dictum mauris nec nunc varius dapibus. Suspendisse venenatis ante maximus leo ornare tincidunt. Nunc vulputate eget orci vel facilisis. Aenean vitae viverra quam, ut porta dui. Donec volutpat ex odio, ut malesuada nulla mattis at. Mauris elementum, nulla eu pharetra consectetur, tellus libero sagittis ligula, id fringilla lectus purus id purus. Suspendisse et felis diam. Interdum et malesuada fames ac ante ipsum primis in faucibus. Proin vitae mi vel sapien fermentum fermentum sed nec odio.
      </div>
    </div>
  </div>
  <div
    className="item"
  >
    <img
      alt="Lorem Ipsum"
      src="https://placehold.it/1920x1080"
    />
    <div
      className="copy"
    >
      <div
        className="title"
      >
        Lorem Ipsum
      </div>
      <div
        className="content"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed lectus lacus, maximus nec ornare a, volutpat vel lorem. Duis erat ex, placerat in cursus vel, posuere ac dui. Vivamus placerat lorem felis, vel tempor leo imperdiet nec. Mauris purus purus, pharetra vitae malesuada vel, mollis sit amet odio. Nam tincidunt nisl ac turpis scelerisque, sed facilisis dui fermentum. Aliquam fermentum cursus mi ut euismod. Phasellus vel convallis orci, vel tempor elit. Vivamus ut ipsum elit. Ut sollicitudin scelerisque ex.Morbi suscipit, nisl id fermentum consectetur, est risus condimentum nulla, bibendum faucibus augue risus ac odio. In a urna tincidunt, scelerisque diam a, dapibus arcu. Proin consequat lectus nec orci venenatis, vel vestibulum dolor rutrum. In velit felis, volutpat eu suscipit eget, congue at ipsum. Integer libero urna, pretium id pretium ac, rhoncus in arcu. Integer vestibulum erat sed dui aliquam, ac gravida neque facilisis. Donec mattis ligula nibh, ac vestibulum dolor consequat ac.Fusce ultricies enim in cursus scelerisque. Pellentesque ut augue eu erat volutpat finibus. Sed at malesuada arcu, id hendrerit lorem. Nulla accumsan vitae sapien at vestibulum. Cras luctus accumsan commodo. Praesent dictum mauris nec nunc varius dapibus. Suspendisse venenatis ante maximus leo ornare tincidunt. Nunc vulputate eget orci vel facilisis. Aenean vitae viverra quam, ut porta dui. Donec volutpat ex odio, ut malesuada nulla mattis at. Mauris elementum, nulla eu pharetra consectetur, tellus libero sagittis ligula, id fringilla lectus purus id purus. Suspendisse et felis diam. Interdum et malesuada fames ac ante ipsum primis in faucibus. Proin vitae mi vel sapien fermentum fermentum sed nec odio.
      </div>
    </div>
  </div>
  <div
    className="item"
  >
    <img
      alt="Lorem Ipsum"
      src="https://placehold.it/1920x1080"
    />
    <div
      className="copy"
    >
      <div
        className="title"
      >
        Lorem Ipsum
      </div>
      <div
        className="content"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed lectus lacus, maximus nec ornare a, volutpat vel lorem. Duis erat ex, placerat in cursus vel, posuere ac dui. Vivamus placerat lorem felis, vel tempor leo imperdiet nec. Mauris purus purus, pharetra vitae malesuada vel, mollis sit amet odio. Nam tincidunt nisl ac turpis scelerisque, sed facilisis dui fermentum. Aliquam fermentum cursus mi ut euismod. Phasellus vel convallis orci, vel tempor elit. Vivamus ut ipsum elit. Ut sollicitudin scelerisque ex.Morbi suscipit, nisl id fermentum consectetur, est risus condimentum nulla, bibendum faucibus augue risus ac odio. In a urna tincidunt, scelerisque diam a, dapibus arcu. Proin consequat lectus nec orci venenatis, vel vestibulum dolor rutrum. In velit felis, volutpat eu suscipit eget, congue at ipsum. Integer libero urna, pretium id pretium ac, rhoncus in arcu. Integer vestibulum erat sed dui aliquam, ac gravida neque facilisis. Donec mattis ligula nibh, ac vestibulum dolor consequat ac.Fusce ultricies enim in cursus scelerisque. Pellentesque ut augue eu erat volutpat finibus. Sed at malesuada arcu, id hendrerit lorem. Nulla accumsan vitae sapien at vestibulum. Cras luctus accumsan commodo. Praesent dictum mauris nec nunc varius dapibus. Suspendisse venenatis ante maximus leo ornare tincidunt. Nunc vulputate eget orci vel facilisis. Aenean vitae viverra quam, ut porta dui. Donec volutpat ex odio, ut malesuada nulla mattis at. Mauris elementum, nulla eu pharetra consectetur, tellus libero sagittis ligula, id fringilla lectus purus id purus. Suspendisse et felis diam. Interdum et malesuada fames ac ante ipsum primis in faucibus. Proin vitae mi vel sapien fermentum fermentum sed nec odio.
      </div>
    </div>
  </div>
  <div
    className="item"
  >
    <img
      alt="Lorem Ipsum"
      src="https://placehold.it/1920x1080"
    />
    <div
      className="copy"
    >
      <div
        className="title"
      >
        Lorem Ipsum
      </div>
      <div
        className="content"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed lectus lacus, maximus nec ornare a, volutpat vel lorem. Duis erat ex, placerat in cursus vel, posuere ac dui. Vivamus placerat lorem felis, vel tempor leo imperdiet nec. Mauris purus purus, pharetra vitae malesuada vel, mollis sit amet odio. Nam tincidunt nisl ac turpis scelerisque, sed facilisis dui fermentum. Aliquam fermentum cursus mi ut euismod. Phasellus vel convallis orci, vel tempor elit. Vivamus ut ipsum elit. Ut sollicitudin scelerisque ex.Morbi suscipit, nisl id fermentum consectetur, est risus condimentum nulla, bibendum faucibus augue risus ac odio. In a urna tincidunt, scelerisque diam a, dapibus arcu. Proin consequat lectus nec orci venenatis, vel vestibulum dolor rutrum. In velit felis, volutpat eu suscipit eget, congue at ipsum. Integer libero urna, pretium id pretium ac, rhoncus in arcu. Integer vestibulum erat sed dui aliquam, ac gravida neque facilisis. Donec mattis ligula nibh, ac vestibulum dolor consequat ac.Fusce ultricies enim in cursus scelerisque. Pellentesque ut augue eu erat volutpat finibus. Sed at malesuada arcu, id hendrerit lorem. Nulla accumsan vitae sapien at vestibulum. Cras luctus accumsan commodo. Praesent dictum mauris nec nunc varius dapibus. Suspendisse venenatis ante maximus leo ornare tincidunt. Nunc vulputate eget orci vel facilisis. Aenean vitae viverra quam, ut porta dui. Donec volutpat ex odio, ut malesuada nulla mattis at. Mauris elementum, nulla eu pharetra consectetur, tellus libero sagittis ligula, id fringilla lectus purus id purus. Suspendisse et felis diam. Interdum et malesuada fames ac ante ipsum primis in faucibus. Proin vitae mi vel sapien fermentum fermentum sed nec odio.
      </div>
    </div>
  </div>
  <div
    className="item"
  >
    <img
      alt="Lorem Ipsum"
      src="https://placehold.it/1920x1080"
    />
    <div
      className="copy"
    >
      <div
        className="title"
      >
        Lorem Ipsum
      </div>
      <div
        className="content"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed lectus lacus, maximus nec ornare a, volutpat vel lorem. Duis erat ex, placerat in cursus vel, posuere ac dui. Vivamus placerat lorem felis, vel tempor leo imperdiet nec. Mauris purus purus, pharetra vitae malesuada vel, mollis sit amet odio. Nam tincidunt nisl ac turpis scelerisque, sed facilisis dui fermentum. Aliquam fermentum cursus mi ut euismod. Phasellus vel convallis orci, vel tempor elit. Vivamus ut ipsum elit. Ut sollicitudin scelerisque ex.Morbi suscipit, nisl id fermentum consectetur, est risus condimentum nulla, bibendum faucibus augue risus ac odio. In a urna tincidunt, scelerisque diam a, dapibus arcu. Proin consequat lectus nec orci venenatis, vel vestibulum dolor rutrum. In velit felis, volutpat eu suscipit eget, congue at ipsum. Integer libero urna, pretium id pretium ac, rhoncus in arcu. Integer vestibulum erat sed dui aliquam, ac gravida neque facilisis. Donec mattis ligula nibh, ac vestibulum dolor consequat ac.Fusce ultricies enim in cursus scelerisque. Pellentesque ut augue eu erat volutpat finibus. Sed at malesuada arcu, id hendrerit lorem. Nulla accumsan vitae sapien at vestibulum. Cras luctus accumsan commodo. Praesent dictum mauris nec nunc varius dapibus. Suspendisse venenatis ante maximus leo ornare tincidunt. Nunc vulputate eget orci vel facilisis. Aenean vitae viverra quam, ut porta dui. Donec volutpat ex odio, ut malesuada nulla mattis at. Mauris elementum, nulla eu pharetra consectetur, tellus libero sagittis ligula, id fringilla lectus purus id purus. Suspendisse et felis diam. Interdum et malesuada fames ac ante ipsum primis in faucibus. Proin vitae mi vel sapien fermentum fermentum sed nec odio.
      </div>
    </div>
  </div>
  <div
    className="item"
  >
    <img
      alt="Lorem Ipsum"
      src="https://placehold.it/1920x1080"
    />
    <div
      className="copy"
    >
      <div
        className="title"
      >
        Lorem Ipsum
      </div>
      <div
        className="content"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed lectus lacus, maximus nec ornare a, volutpat vel lorem. Duis erat ex, placerat in cursus vel, posuere ac dui. Vivamus placerat lorem felis, vel tempor leo imperdiet nec. Mauris purus purus, pharetra vitae malesuada vel, mollis sit amet odio. Nam tincidunt nisl ac turpis scelerisque, sed facilisis dui fermentum. Aliquam fermentum cursus mi ut euismod. Phasellus vel convallis orci, vel tempor elit. Vivamus ut ipsum elit. Ut sollicitudin scelerisque ex.Morbi suscipit, nisl id fermentum consectetur, est risus condimentum nulla, bibendum faucibus augue risus ac odio. In a urna tincidunt, scelerisque diam a, dapibus arcu. Proin consequat lectus nec orci venenatis, vel vestibulum dolor rutrum. In velit felis, volutpat eu suscipit eget, congue at ipsum. Integer libero urna, pretium id pretium ac, rhoncus in arcu. Integer vestibulum erat sed dui aliquam, ac gravida neque facilisis. Donec mattis ligula nibh, ac vestibulum dolor consequat ac.Fusce ultricies enim in cursus scelerisque. Pellentesque ut augue eu erat volutpat finibus. Sed at malesuada arcu, id hendrerit lorem. Nulla accumsan vitae sapien at vestibulum. Cras luctus accumsan commodo. Praesent dictum mauris nec nunc varius dapibus. Suspendisse venenatis ante maximus leo ornare tincidunt. Nunc vulputate eget orci vel facilisis. Aenean vitae viverra quam, ut porta dui. Donec volutpat ex odio, ut malesuada nulla mattis at. Mauris elementum, nulla eu pharetra consectetur, tellus libero sagittis ligula, id fringilla lectus purus id purus. Suspendisse et felis diam. Interdum et malesuada fames ac ante ipsum primis in faucibus. Proin vitae mi vel sapien fermentum fermentum sed nec odio.
      </div>
    </div>
  </div>
  <div
    className="item"
  >
    <img
      alt="Lorem Ipsum"
      src="https://placehold.it/1920x1080"
    />
    <div
      className="copy"
    >
      <div
        className="title"
      >
        Lorem Ipsum
      </div>
      <div
        className="content"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed lectus lacus, maximus nec ornare a, volutpat vel lorem. Duis erat ex, placerat in cursus vel, posuere ac dui. Vivamus placerat lorem felis, vel tempor leo imperdiet nec. Mauris purus purus, pharetra vitae malesuada vel, mollis sit amet odio. Nam tincidunt nisl ac turpis scelerisque, sed facilisis dui fermentum. Aliquam fermentum cursus mi ut euismod. Phasellus vel convallis orci, vel tempor elit. Vivamus ut ipsum elit. Ut sollicitudin scelerisque ex.Morbi suscipit, nisl id fermentum consectetur, est risus condimentum nulla, bibendum faucibus augue risus ac odio. In a urna tincidunt, scelerisque diam a, dapibus arcu. Proin consequat lectus nec orci venenatis, vel vestibulum dolor rutrum. In velit felis, volutpat eu suscipit eget, congue at ipsum. Integer libero urna, pretium id pretium ac, rhoncus in arcu. Integer vestibulum erat sed dui aliquam, ac gravida neque facilisis. Donec mattis ligula nibh, ac vestibulum dolor consequat ac.Fusce ultricies enim in cursus scelerisque. Pellentesque ut augue eu erat volutpat finibus. Sed at malesuada arcu, id hendrerit lorem. Nulla accumsan vitae sapien at vestibulum. Cras luctus accumsan commodo. Praesent dictum mauris nec nunc varius dapibus. Suspendisse venenatis ante maximus leo ornare tincidunt. Nunc vulputate eget orci vel facilisis. Aenean vitae viverra quam, ut porta dui. Donec volutpat ex odio, ut malesuada nulla mattis at. Mauris elementum, nulla eu pharetra consectetur, tellus libero sagittis ligula, id fringilla lectus purus id purus. Suspendisse et felis diam. Interdum et malesuada fames ac ante ipsum primis in faucibus. Proin vitae mi vel sapien fermentum fermentum sed nec odio.
      </div>
    </div>
  </div>
  <div
    className="item"
  >
    <img
      alt="Lorem Ipsum"
      src="https://placehold.it/1920x1080"
    />
    <div
      className="copy"
    >
      <div
        className="title"
      >
        Lorem Ipsum
      </div>
      <div
        className="content"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed lectus lacus, maximus nec ornare a, volutpat vel lorem. Duis erat ex, placerat in cursus vel, posuere ac dui. Vivamus placerat lorem felis, vel tempor leo imperdiet nec. Mauris purus purus, pharetra vitae malesuada vel, mollis sit amet odio. Nam tincidunt nisl ac turpis scelerisque, sed facilisis dui fermentum. Aliquam fermentum cursus mi ut euismod. Phasellus vel convallis orci, vel tempor elit. Vivamus ut ipsum elit. Ut sollicitudin scelerisque ex.Morbi suscipit, nisl id fermentum consectetur, est risus condimentum nulla, bibendum faucibus augue risus ac odio. In a urna tincidunt, scelerisque diam a, dapibus arcu. Proin consequat lectus nec orci venenatis, vel vestibulum dolor rutrum. In velit felis, volutpat eu suscipit eget, congue at ipsum. Integer libero urna, pretium id pretium ac, rhoncus in arcu. Integer vestibulum erat sed dui aliquam, ac gravida neque facilisis. Donec mattis ligula nibh, ac vestibulum dolor consequat ac.Fusce ultricies enim in cursus scelerisque. Pellentesque ut augue eu erat volutpat finibus. Sed at malesuada arcu, id hendrerit lorem. Nulla accumsan vitae sapien at vestibulum. Cras luctus accumsan commodo. Praesent dictum mauris nec nunc varius dapibus. Suspendisse venenatis ante maximus leo ornare tincidunt. Nunc vulputate eget orci vel facilisis. Aenean vitae viverra quam, ut porta dui. Donec volutpat ex odio, ut malesuada nulla mattis at. Mauris elementum, nulla eu pharetra consectetur, tellus libero sagittis ligula, id fringilla lectus purus id purus. Suspendisse et felis diam. Interdum et malesuada fames ac ante ipsum primis in faucibus. Proin vitae mi vel sapien fermentum fermentum sed nec odio.
      </div>
    </div>
  </div>
  <div
    className="item"
  >
    <img
      alt="Lorem Ipsum"
      src="https://placehold.it/1920x1080"
    />
    <div
      className="copy"
    >
      <div
        className="title"
      >
        Lorem Ipsum
      </div>
      <div
        className="content"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed lectus lacus, maximus nec ornare a, volutpat vel lorem. Duis erat ex, placerat in cursus vel, posuere ac dui. Vivamus placerat lorem felis, vel tempor leo imperdiet nec. Mauris purus purus, pharetra vitae malesuada vel, mollis sit amet odio. Nam tincidunt nisl ac turpis scelerisque, sed facilisis dui fermentum. Aliquam fermentum cursus mi ut euismod. Phasellus vel convallis orci, vel tempor elit. Vivamus ut ipsum elit. Ut sollicitudin scelerisque ex.Morbi suscipit, nisl id fermentum consectetur, est risus condimentum nulla, bibendum faucibus augue risus ac odio. In a urna tincidunt, scelerisque diam a, dapibus arcu. Proin consequat lectus nec orci venenatis, vel vestibulum dolor rutrum. In velit felis, volutpat eu suscipit eget, congue at ipsum. Integer libero urna, pretium id pretium ac, rhoncus in arcu. Integer vestibulum erat sed dui aliquam, ac gravida neque facilisis. Donec mattis ligula nibh, ac vestibulum dolor consequat ac.Fusce ultricies enim in cursus scelerisque. Pellentesque ut augue eu erat volutpat finibus. Sed at malesuada arcu, id hendrerit lorem. Nulla accumsan vitae sapien at vestibulum. Cras luctus accumsan commodo. Praesent dictum mauris nec nunc varius dapibus. Suspendisse venenatis ante maximus leo ornare tincidunt. Nunc vulputate eget orci vel facilisis. Aenean vitae viverra quam, ut porta dui. Donec volutpat ex odio, ut malesuada nulla mattis at. Mauris elementum, nulla eu pharetra consectetur, tellus libero sagittis ligula, id fringilla lectus purus id purus. Suspendisse et felis diam. Interdum et malesuada fames ac ante ipsum primis in faucibus. Proin vitae mi vel sapien fermentum fermentum sed nec odio.
      </div>
    </div>
  </div>
  <div
    className="item"
  >
    <img
      alt="Lorem Ipsum"
      src="https://placehold.it/1920x1080"
    />
    <div
      className="copy"
    >
      <div
        className="title"
      >
        Lorem Ipsum
      </div>
      <div
        className="content"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed lectus lacus, maximus nec ornare a, volutpat vel lorem. Duis erat ex, placerat in cursus vel, posuere ac dui. Vivamus placerat lorem felis, vel tempor leo imperdiet nec. Mauris purus purus, pharetra vitae malesuada vel, mollis sit amet odio. Nam tincidunt nisl ac turpis scelerisque, sed facilisis dui fermentum. Aliquam fermentum cursus mi ut euismod. Phasellus vel convallis orci, vel tempor elit. Vivamus ut ipsum elit. Ut sollicitudin scelerisque ex.Morbi suscipit, nisl id fermentum consectetur, est risus condimentum nulla, bibendum faucibus augue risus ac odio. In a urna tincidunt, scelerisque diam a, dapibus arcu. Proin consequat lectus nec orci venenatis, vel vestibulum dolor rutrum. In velit felis, volutpat eu suscipit eget, congue at ipsum. Integer libero urna, pretium id pretium ac, rhoncus in arcu. Integer vestibulum erat sed dui aliquam, ac gravida neque facilisis. Donec mattis ligula nibh, ac vestibulum dolor consequat ac.Fusce ultricies enim in cursus scelerisque. Pellentesque ut augue eu erat volutpat finibus. Sed at malesuada arcu, id hendrerit lorem. Nulla accumsan vitae sapien at vestibulum. Cras luctus accumsan commodo. Praesent dictum mauris nec nunc varius dapibus. Suspendisse venenatis ante maximus leo ornare tincidunt. Nunc vulputate eget orci vel facilisis. Aenean vitae viverra quam, ut porta dui. Donec volutpat ex odio, ut malesuada nulla mattis at. Mauris elementum, nulla eu pharetra consectetur, tellus libero sagittis ligula, id fringilla lectus purus id purus. Suspendisse et felis diam. Interdum et malesuada fames ac ante ipsum primis in faucibus. Proin vitae mi vel sapien fermentum fermentum sed nec odio.
      </div>
    </div>
  </div>
  <div
    className="item"
  >
    <img
      alt="Lorem Ipsum"
      src="https://placehold.it/1920x1080"
    />
    <div
      className="copy"
    >
      <div
        className="title"
      >
        Lorem Ipsum
      </div>
      <div
        className="content"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed lectus lacus, maximus nec ornare a, volutpat vel lorem. Duis erat ex, placerat in cursus vel, posuere ac dui. Vivamus placerat lorem felis, vel tempor leo imperdiet nec. Mauris purus purus, pharetra vitae malesuada vel, mollis sit amet odio. Nam tincidunt nisl ac turpis scelerisque, sed facilisis dui fermentum. Aliquam fermentum cursus mi ut euismod. Phasellus vel convallis orci, vel tempor elit. Vivamus ut ipsum elit. Ut sollicitudin scelerisque ex.Morbi suscipit, nisl id fermentum consectetur, est risus condimentum nulla, bibendum faucibus augue risus ac odio. In a urna tincidunt, scelerisque diam a, dapibus arcu. Proin consequat lectus nec orci venenatis, vel vestibulum dolor rutrum. In velit felis, volutpat eu suscipit eget, congue at ipsum. Integer libero urna, pretium id pretium ac, rhoncus in arcu. Integer vestibulum erat sed dui aliquam, ac gravida neque facilisis. Donec mattis ligula nibh, ac vestibulum dolor consequat ac.Fusce ultricies enim in cursus scelerisque. Pellentesque ut augue eu erat volutpat finibus. Sed at malesuada arcu, id hendrerit lorem. Nulla accumsan vitae sapien at vestibulum. Cras luctus accumsan commodo. Praesent dictum mauris nec nunc varius dapibus. Suspendisse venenatis ante maximus leo ornare tincidunt. Nunc vulputate eget orci vel facilisis. Aenean vitae viverra quam, ut porta dui. Donec volutpat ex odio, ut malesuada nulla mattis at. Mauris elementum, nulla eu pharetra consectetur, tellus libero sagittis ligula, id fringilla lectus purus id purus. Suspendisse et felis diam. Interdum et malesuada fames ac ante ipsum primis in faucibus. Proin vitae mi vel sapien fermentum fermentum sed nec odio.
      </div>
    </div>
  </div>
  <div
    className="item"
  >
    <img
      alt="Lorem Ipsum"
      src="https://placehold.it/1920x1080"
    />
    <div
      className="copy"
    >
      <div
        className="title"
      >
        Lorem Ipsum
      </div>
      <div
        className="content"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed lectus lacus, maximus nec ornare a, volutpat vel lorem. Duis erat ex, placerat in cursus vel, posuere ac dui. Vivamus placerat lorem felis, vel tempor leo imperdiet nec. Mauris purus purus, pharetra vitae malesuada vel, mollis sit amet odio. Nam tincidunt nisl ac turpis scelerisque, sed facilisis dui fermentum. Aliquam fermentum cursus mi ut euismod. Phasellus vel convallis orci, vel tempor elit. Vivamus ut ipsum elit. Ut sollicitudin scelerisque ex.Morbi suscipit, nisl id fermentum consectetur, est risus condimentum nulla, bibendum faucibus augue risus ac odio. In a urna tincidunt, scelerisque diam a, dapibus arcu. Proin consequat lectus nec orci venenatis, vel vestibulum dolor rutrum. In velit felis, volutpat eu suscipit eget, congue at ipsum. Integer libero urna, pretium id pretium ac, rhoncus in arcu. Integer vestibulum erat sed dui aliquam, ac gravida neque facilisis. Donec mattis ligula nibh, ac vestibulum dolor consequat ac.Fusce ultricies enim in cursus scelerisque. Pellentesque ut augue eu erat volutpat finibus. Sed at malesuada arcu, id hendrerit lorem. Nulla accumsan vitae sapien at vestibulum. Cras luctus accumsan commodo. Praesent dictum mauris nec nunc varius dapibus. Suspendisse venenatis ante maximus leo ornare tincidunt. Nunc vulputate eget orci vel facilisis. Aenean vitae viverra quam, ut porta dui. Donec volutpat ex odio, ut malesuada nulla mattis at. Mauris elementum, nulla eu pharetra consectetur, tellus libero sagittis ligula, id fringilla lectus purus id purus. Suspendisse et felis diam. Interdum et malesuada fames ac ante ipsum primis in faucibus. Proin vitae mi vel sapien fermentum fermentum sed nec odio.
      </div>
    </div>
  </div>
  <div
    className="item"
  >
    <img
      alt="Lorem Ipsum"
      src="https://placehold.it/1920x1080"
    />
    <div
      className="copy"
    >
      <div
        className="title"
      >
        Lorem Ipsum
      </div>
      <div
        className="content"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed lectus lacus, maximus nec ornare a, volutpat vel lorem. Duis erat ex, placerat in cursus vel, posuere ac dui. Vivamus placerat lorem felis, vel tempor leo imperdiet nec. Mauris purus purus, pharetra vitae malesuada vel, mollis sit amet odio. Nam tincidunt nisl ac turpis scelerisque, sed facilisis dui fermentum. Aliquam fermentum cursus mi ut euismod. Phasellus vel convallis orci, vel tempor elit. Vivamus ut ipsum elit. Ut sollicitudin scelerisque ex.Morbi suscipit, nisl id fermentum consectetur, est risus condimentum nulla, bibendum faucibus augue risus ac odio. In a urna tincidunt, scelerisque diam a, dapibus arcu. Proin consequat lectus nec orci venenatis, vel vestibulum dolor rutrum. In velit felis, volutpat eu suscipit eget, congue at ipsum. Integer libero urna, pretium id pretium ac, rhoncus in arcu. Integer vestibulum erat sed dui aliquam, ac gravida neque facilisis. Donec mattis ligula nibh, ac vestibulum dolor consequat ac.Fusce ultricies enim in cursus scelerisque. Pellentesque ut augue eu erat volutpat finibus. Sed at malesuada arcu, id hendrerit lorem. Nulla accumsan vitae sapien at vestibulum. Cras luctus accumsan commodo. Praesent dictum mauris nec nunc varius dapibus. Suspendisse venenatis ante maximus leo ornare tincidunt. Nunc vulputate eget orci vel facilisis. Aenean vitae viverra quam, ut porta dui. Donec volutpat ex odio, ut malesuada nulla mattis at. Mauris elementum, nulla eu pharetra consectetur, tellus libero sagittis ligula, id fringilla lectus purus id purus. Suspendisse et felis diam. Interdum et malesuada fames ac ante ipsum primis in faucibus. Proin vitae mi vel sapien fermentum fermentum sed nec odio.
      </div>
    </div>
  </div>
</div>
`;
